.github/workflows/loadtest.yaml

```diff
@@ -0,0 +1,39 @@
+name: k6 test
+on: 
+  workflow_dispatch:
+    inputs:
+      logLevel:
+        description: 'Log level'     
+        required: true
+        default: 'warning'
+      tags:
+        description: 'K6 Load Test'  
+
+jobs:
+  k6_test_run:
+    runs-on: ubuntu-latest
+
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v1
+        with:
+          repository: longwuyuan/ingress-nginx
+          ref: k6
+
+      - name: Create cluster
+        continue-on-error: true
+        run: |
+          mkdir $HOME/.kube
+          which kubectl
+          which kind
+          make dev-env
+          sleep 60
+          kubectl get po -A
+          kubectl create deploy k6 --image nginx:alpine --port 80 && kubectl expose deploy k6 --port 80 && kubectl create ing k6 --class nginx --rule localhost/=k6:80
+          sleep 15
+          kubectl get all,ing
+          curl localhost
+ 
+      - name: Run k6 test
+        run: |
+          ./test/k6/k6 run ./test/k6/k6.test.js
```

Comment:
Can we download k6 instead of ship it as part of the code?

Reply:
If you prefer, we can change to download. But I already played with that option and found version is locked (no "latest") so 3 extra lines of code and will have to keep updating if link breaks . Please confirm and I will change to download if you decide.

Reply:
Changed to download. @rikatz please check. Seek ing approve so can see first real optional execution of test